### ISSUE TITLE
Please create at least a draft pass at WCAG2ICT-ifying the new SCs

### ISSUE THREAD

**Issue by peterkorn** [w3c/wcag21#752]
We can anticipate organizations attempting to apply WCAG 2.1 to non-web ICT, just as we saw groups do that with v2.0.  v2.1 introduces a number of new SCs, some of which may not fit well for non-web ICT (such as 1.4.13 and 2.5.2 which could use an exception for AT like Dasher; 2.4.11 and 2.5.1 which may not be necessary on platforms with built-in AT and where the Understanding text doesn't seem to anticipate speaker ID in voice interfaces which can reject commands not coming from the identified speaker).

Failing such a draft, please consider at a minimum issuing a clear statement from the W3C/WAI (or at least WCAG WG) stating that pending an update to WCAG2ICT, they recommend NOT applying anything beyond v2.0 to non-Web ICT.

**Comment by awkawk** [comment]
Thank you for commenting. For more information about how the AG WG will process this issue, see the following:
- [Issue Processing](https://www.w3.org/WAI/WCAG21/comments/#issueprocessing) - How the WG will discuss the issue
- [Issue Resolution](https://www.w3.org/WAI/WCAG21/comments/#issueresolution) - Identifying and responding to the official WG response
- [Issue Notification](https://www.w3.org/WAI/WCAG21/comments/#issuenotification) - How to limit notifications from GitHub

**Comment by joshueoconnor** [comment]
Thanks @peterkorn - good idea. Are you happy to look into this and suggest some text? It may be straightforward, but on the same hand may not.

**Comment by DavidMacDonald** [comment]
I've created a preliminary discussion draft of WCAG2ICT for the new SCs.
http://davidmacd.com/WCAG/wcag2ict-21-discussion.html

**Comment by awkawk** [comment]
[Proposed WG Response]
Thank you for the comment. The Working Group did make efforts to make the new SC technology-independent but agree that more in-depth study is warranted in the form of an update to WCAG2ICT.

The Working Group will review this question for the WCAG 2.1 FAQ, but this question is somewhat separate from the normative WCAG 2.1 document development, so we do not believe any changes to the WCAG 2.1 document are required at this time.

**Comment by awkawk** [comment]
The WG decided on the above response, so we changed the text in the comment containing the proposed response to read "[Official WG Response]". Please confirm is you are satisfied with the response within 3 days. If we haven't heard a response by then we will regard the resolution as satisfactory.

**Comment by awkawk** [comment]
Leaving open for FAQ changes

**Comment by michael-n-cooper** [comment]
Referencing #866 for actioning.
